pkg/cmd/root.go

```diff
@@ -7,13 +7,15 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const Version = "0.0.1-dev"
+
 // RootCmd is the top command, other commands should be its child
 var RootCmd = &cobra.Command{
 	Use:   "xephon-b",
 	Short: "Time series benmark suite",
 	Long:  `Xephon-B is a benmark suite with a micro benmark tool`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("Hello, Ayato")
```

Comment:
btw: @czheo does [Ayato](http://rahxephon.wikia.com/wiki/Ayato_Kamina) has any special meaning in Japanese, or it's just some word that the author created.